src/Core/RulesActionBase.php

```diff
@@ -31,6 +32,20 @@
   /**
    * {@inheritdoc}
    */
+  public function getContextValue($name) {
```

Comment:
Added this as workarund for https://www.drupal.org/node/2677162 - so people do not have to.